packages/next-swc/crates/napi/src/turbo_trace_server.rs

```diff
@@ -0,0 +1,8 @@
+use std::path::PathBuf;
+
+#[napi]
+pub async fn start_turbopack_trace_server(path: String) -> napi::Result<String> {
+    let path_buf = PathBuf::from(path);
+    turbopack_binding::turbopack::trace_server::start_turbopack_trace_server(path_buf);
```

Comment:
I think this would permanently lock up a node thread pool thread. Better wrap that in a `thread::spawn` to give it a separate thread.

Reply:
Great, that's what I was looking for in order to make it resolve still